### DISCUSSION TITLE
Starting the NixOS-WSL distribution seems to break the ability to use windows executables from all other non-NixOS wsl distros

### DISCUSSION THREAD

**Discussion by e-nikolov** [nix-community/NixOS-WSL#91] [Q&A]
I'm not sure if this is a bug in NixOS-WSL, a bug in a dependency or if I've done something wrong. 
I already had a Ubuntu based WSL distro and then I imported the NixOS-WSL one. 

If I start the Ubuntu based distro alone, I can run Windows executables from it, e.g., `explorer.exe` opens an explorer window.
When I start the NixOS distro as well, running Windows executables from it works fine, but my Ubuntu based distro starts returning an error:  
```
❯ explorer.exe
explorer.exe: command not found
```

I played around with it a bit and added 1 more NixOS distro and 1 more Ubuntu distro. Starting them all up results in both NixOS distros working fine and both Ubuntu based distros being unable to run `explorer.exe`

Any idea what might be going on?

**Comment by claes** [comment]
Looks related? https://github.com/nix-community/NixOS-WSL/discussions/66

**Reply by nzbr** [reply]
Yes, it's a side effect of #64. You need to set `wsl.interop.register = false`, `nixos-rebuild` and restart the WSL VM with `wsl --shutdown`. That should fix it
